nflow-engine/src/main/java/io/nflow/engine/workflow/curated/CronWorkflow.java

```diff
@@ -97,7 +103,12 @@ protected CronWorkflow(String type, WorkflowSettings settings) {
    * <pre>
    * public NextAction doWork(StateExecution execution) {
    *   // do the work here
-   *   return NextAction.moveToState(schedule, "Work done");
+   *   if (waitForChildWorkflowsToFinish) {
```

Comment:
Wrong condition in the example? Better condition below?

```
if (rescheduleImmediately) {
  return NextAction.moveToState(schedule, "Work done");
} else {
  return NextAction.moveToStateAfter(waitForWorkToFinish, DateTime.now().plusHours(hoursToWait),
    "Waiting for work to finish");
}
```

Reply:
Fixed